### PR TITLE
Fix hidden case for FocusableElementNotVisibleNotAriaHidden

### PR DESCRIPTION
**Pull request by alice**
@ckundo PTAL

### CODE REVIEW

File: src/audits/FocusableElementNotVisibleAndNotAriaHidden.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ axs.AuditRules.addRule({
         }
         // Ignore elements which have a negative tabindex and no text content,
         // as they will be skipped by assistive technology
-        if (axs.properties.findTextAlternatives(element, {}).trim() === '')
+        var textAlternatives = axs.properties.findTextAlternatives(element, {});
+        if (!textAlternatives || textAlternatives.trim() === '')
```
Comment (ckundo) [inline src/audits/FocusableElementNotVisibleAndNotAriaHidden.js]: I think we should add `===` as a best practice to the yet-to-be styleguide.

**Comment by ckundo** [comment]
LGTM! Was the original test passing because it was not running in the browser?

**Comment by alice** [comment]
The original test wasn't running at all because it wasn't included in the non-browser tests.

File: src/audits/FocusableElementNotVisibleAndNotAriaHidden.js (hunk 1 above)
Comment (alice) [inline src/audits/FocusableElementNotVisibleAndNotAriaHidden.js]: Sounds good :smile:

**Pull request merged**